libraries/csharp-net22-runtime/context/setup

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
we don't need a setup script for new languages - don't see this in use anywhere

Reply:
How are the `Pipe` artefacts constructed then?